src/easymotion/easymotion.ts

```diff
@@ -299,18 +301,12 @@ export class EasyMotion {
     this.visibleMarkers = [];
     this.decorations = [];
 
-    const fontFamily = vscode.workspace.getConfiguration().get("editor.fontFamily") as string;
-    const fontSize = vscode.workspace.getConfiguration().get("editor.fontSize") as string;
-    const fontWeight = vscode.workspace.getConfiguration().get("editor.fontWeight") as string;
+    const fontFamily = Configuration.easymotionMarkerFontFamily;
+    const fontSize = Configuration.easymotionMarkerFontSize;
+    const fontWeight = Configuration.easymotionMarkerFontWeight;
 
     // Compute font color based on background (remove opacity)
-    var Color = require('color');
-    let backgroundColor = Color.rgb(Configuration.searchHighlightColor).alpha(1.0);
```

Comment:
Can you remove this dependency from package.json since this was the only place it was used